tests/integrations/django/test_cache_module.py

```diff
@@ -536,12 +536,13 @@ def test_cache_spans_get_many(
         cache.get_many([f"S{id}", f"S{id+1}"])
 
     (transaction,) = events
+    assert transaction["transaction"] == "caches"
     assert len(transaction["spans"]) == 7
 
     assert (
         render_span_tree(transaction)
         == f"""\
-- op="caches": description=null
```

Comment:
here and in all subsequent tests, I made it clear that the transaction name is correct and that the op of the root span is not the transaction name anymore